build/run-me.sh

```diff
@@ -154,6 +171,14 @@ while [ $# -gt 0 ]; do
     opt_append=inject_mcc_files
     ;;
 
+  --kernel-rot-crt)
```

Comment:
I don't like this parameter name.
I think we should use `fit` instead of `kernel` since the filename of the key has `fit` in it.

Reply:
I'll defer to @jjohnson-arm on the question of what the parameter name should be (he specified it in ARMmbed/mbl-docs#107).

I don't agree with the reasoning about the filename though - I think we should just choose the best name for the parameter, and if it makes sense for the filename to match then we should change the filename.

Reply:
The certificate and the key are used to sign the FIT image. Or am I missing something else?
And its not only the kernel, we have the device-tree binary and initramfs bundled all together.

Reply:
Yeah, we've decided to call that chunk of stuff (the FIT) the "Kernel" update component though, so this is consistent with that.

Reply:
This naming convention is rather confusing and at some point, we will have to address this and find some consistency.
The FIT image (containing u-boot script, Linux Kernel, Linux Device Tree and Initiramfs) is flashed in a partition labeled as `boot`. The update component that includes the FIT image is called `kernel` (why???) as well as the parameter used to pass the key/certificate, but the key filename used to sign it includes `fit` in the name.

Reply:
I remember @jonathanhaigh-arm @jjohnson-arm and I had a long conversation about the naming of the `kernel` update component (as I had initially named the parameter in `create-update-payload` as `--fit`) and it was decided in that conversation to name it `--kernel`. We probably need to discuss this again and review the naming convention if it's too confusing.

Reply:
Personally, I think "fit" would be too confusing. A user shouldn't have to know about how we package up the kernel and related things. They shouldn't have to know what "fit" means.

Reply:
Neither the name `kernel` reflects what it is in reality. How the user will know that he is updating/signing/securing other components like dtb and initiramfs?

If the foundation of the project for the secure boot implementation implies splitting the different boot components into different images we should educate the users what these different images are, what are their contents and names. We even have extensive documentation about the boot flow system architecture.